src/index.ts

```diff
@@ -15,6 +15,9 @@ import { App } from "./app/App";
 const app: App = new App();
 export default app;
 
+import * as Enums from './enums';
```

Comment:
@oopsidoodles care to explain how this resolves the issue you were talking to me yesterday about ?

Reply:
@bholagabbar enums.ts is not being committed, when you clone and run install, npm automatically calls build right after, and so the enums file will be generated and this will work.